packages/server/lib/modes/run.js

```diff
@@ -119,7 +126,15 @@ const formatSymbolSummary = (failures) => {
   return getSymbol(failures)
 }
 
-const formatPath = (name, n, colour = 'reset') => {
+const macOSRemovePrivate = (str) => {
+  if (str.startsWith('/private')) {
```

Comment:
Is it possible that we're not on a mac and legitimately have a directory that starts with private?

Reply:
Sure https://github.com/cypress-io/cypress/pull/19319/commits/dc67b143ed92ac72a306d437dc83a236aa69fb81#diff-58fd4a8e28c13b52b19c42346eade66ed1de5a8b139541dff4e94fef42c3c701R131

Note: this is only used for the system tests; it won't impact general users, it's only called [here](https://github.com/cypress-io/cypress/pull/19319/commits/dc67b143ed92ac72a306d437dc83a236aa69fb81#diff-58fd4a8e28c13b52b19c42346eade66ed1de5a8b139541dff4e94fef42c3c701R143). Either way, I added an additional check.